reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -74,7 +74,7 @@ export default function WorkflowList({ workflows, loading }) {
                 }`}
               >
                 <Icon name="hdd" />
-                {size}
+                {size.human_readable}
```

Comment:
I see that while the workflow is running we still don't have its size but the icon shows anyway.

![image](https://user-images.githubusercontent.com/8863238/97455727-73682480-1938-11eb-95ca-55a0e5c119cb.png)

WDYT about hiding the icon if we don't have the human-readable value?

Reply:
Also, nor necessarily belonging to this PR, but we should display the size on the details page as well, we're not doing it at the moment. This comment can be converted into an issue if you find this appropriate.

Reply:
> I see that while the workflow is running we still don't have its size but the icon shows anyway.

Fixed, thanks!

> Also, not necessarily belonging to this PR, but we should display the size on the details page as well, we're not doing it at the moment. This comment can be converted into an issue if you find this appropriate.

Good catch! Let's do this in another issue, just to not delay this PR group more.